src/Core/ServiceWrapper/Main.cs

```diff
@@ -642,39 +642,21 @@ public static void Run(string[] _args, ServiceDescriptor? descriptor = null)
                     password,
                     descriptor.ServiceDependencies);
 
-                // update the description
-                /* Somehow this doesn't work, even though it doesn't report an error
-                Win32Service s = svc.Select(d.Id);
-                s.Description = d.Description;

```

Comment:
I wonder whether it is still an issue on old systems, but I am happy to remove this hack

Reply:
Do you mean setting description via `Win32_Service`?

Reply:
Yes. I have no practical way to test on .NET Framework 2 now. But we can go ahead and merge it. It is not a critical functionality, so we can restore this hack for old platforms if needed

Reply:
The `Win32_Service` approach doesn't work because `Description` is a read-only property.